backend/src/test/java/com/thoughtworks/aceleradora/controller/CreateSolicitacaoEndToEndTest.java

```diff
@@ -0,0 +1,35 @@
+package com.thoughtworks.aceleradora.controller;
```

Comment:
Sempre temos que pensar se os testes E2E dentro de um backend só realmente são necessários. (Só uma food for thought)

Reply:
Esse teste era temporário não consegui tempo para ajustar os testes de controller então por enquanto removi os testes de controller.